### PR TITLE
feat(angular): Add guide for uploading source maps with `@nx/angular` CLI

### PR DESCRIPTION
**Pull request by Lms24**
Based on our findings from https://github.com/getsentry/sentry-javascript/issues/8267 let's bring this guide to our docs. 

Verified the steps with a simple Nx Angular test app: https://github.com/Lms24/nx-angular-sentry

### CODE REVIEW

**Review by mydea — APPROVED** [review]
sweet! 🚀

**Review by vivianyentran — APPROVED** [review]
Thanks for adding these updates!

File: src/platform-includes/sourcemaps/overview/javascript.angular.mdx [hunk 1]
```diff
@@ -28,12 +28,14 @@ To generate source maps, you need to add the `sourceMap` option to your `angular
 }
 ```
 
-### Two Ways to Upload Source Maps
+### Three Ways to Upload Source Maps
 
 You can either set releases and upload source maps automatically using the Angular CLI, or upload source maps manually using the Sentry-CLI.
```
[the remark this answers is not in the document]
Comment (Lms24) [reply src/platform-includes/sourcemaps/overview/javascript.angular.mdx:33]: +1, good point!

File: src/platform-includes/sourcemaps/upload/primer/javascript.angular.mdx
[the remark this answers is not in the document]
Comment (Lms24) [reply src/platform-includes/sourcemaps/upload/primer/javascript.angular.mdx:1]: Done ;)

**Pull request merged**